---
float64 E_speed
float64 E_headvel
float64 E_headori
float64 E_slide
float64 E_effort